v3/plugins/redhat/vscode-openshift-connector/0.1.2/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-openshift-connector
+version: 0.1.2
+type: VS Code extension
+displayName: OpenShift Connector
+title: OpenShift Connector
+description: Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Eclipse Che
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-openshift-tools
+category: Other
+firstPublicationDate: "2019-10-08"
+spec:
+  containers:
+    - image: "docker.io/eclipse/che-remote-plugin-openshift-connector-0.1.2:next"
```

Comment:
name will be different
it will be published under https://quay.io/repository/eclipse/che-sidecar-openshift-connector?tab=tags

Reply:
like https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/redhat/vscode-openshift-connector/0.1.1/meta.yaml#L15

Reply:
follow-up of https://github.com/eclipse/che/issues/15109

Reply:
the link is updated